build/nuke/Build.Steps.cs

```diff
@@ -350,40 +351,109 @@ partial class Build
+                    .EnableNoBuild()
+                    .EnableNoRestore()
+                    .CombineWith(TestFrameworks.ExceptNetFramework(), (p, framework) => p
+                        .SetFramework(framework)
+                        // Additional-deps probes the directory using SemVer format.
+                        // Example: For netcoreapp3.1 framework, additional-deps uses 3.1.0 or 3.1.1 and so on.
+                        // Major and Minor version are extracted from framework and default value of 0 is appended for patch.
+                        .SetOutput(AdditionalDepsDirectory / "shared" / "Microsoft.NETCore.App" / framework.ToString().Substring(framework.ToString().Length - 3) + ".0")));
+
+
+                AdditionalDepsDirectory.GlobFiles("**/*deps.json")
+                    .ForEach(file =>
+                    {
+                        var depsJsonContent = File.ReadAllText(file);
+                        CopyNativeDependenciesToStore(file, depsJsonContent);
+
+                        RemoveOpenTelemetryAutoInstrumentationAdditionalDepsFromDepsFile(depsJsonContent, file);
+                    });
+                RemoveFilesFromAdditionalDepsDirectory();
+
+                void CopyNativeDependenciesToStore(AbsolutePath file, string depsJsonContent)
+                {
+                    var depsDirectory = file.Parent;
+                    var targetDirectory = Path.Combine(depsDirectory.Parent.Parent.Parent.Parent, "store");
+                    using var jsonDocument = JsonDocument.Parse(depsJsonContent);
+
+                    var runtimeName = jsonDocument.RootElement.GetProperty("runtimeTarget").GetProperty("name").GetString();
+                    var folderRuntimeName = runtimeName switch
+                    {
+                        ".NETCoreApp,Version=v3.1" => "netcoreapp3.1",
+                        ".NETCoreApp,Version=v6.0" => "net6.0",
+                        _ => throw new ArgumentOutOfRangeException(nameof(runtimeName), runtimeName,
+                            "This value is not supported. You have probably introduced new .NET version to AutoInstrumentation")
+                    };
+
+                    foreach (var targetProperty in jsonDocument.RootElement.GetProperty("targets").EnumerateObject())
+                    {
+                        var target = targetProperty.Value;
+
+                        foreach (var packages in target.EnumerateObject())
+                        {
+                            if (!packages.Value.TryGetProperty("runtimeTargets", out var runtimeTargets))
+                            {
+                                continue;
+                            }
+
+                            foreach (var runtimeDependency in runtimeTargets.EnumerateObject())
+                            {
+                                var sourceFileName = Path.Combine(depsDirectory, runtimeDependency.Name);
+
+
```

Comment:
I'm just wondering the stylecop (or else) is allowing it. 
nit: 2 empty lines.

Reply:
Fixed. I suppose that build folder is not included into code style check.